internal/resources/examples_test.go

```diff
@@ -82,6 +82,12 @@ func TestAccExamples(t *testing.T) {
 				testutils.CheckCloudInstanceTestsEnabled(t)
 			},
 		},
+		{
+			category: "Connections",
+			testCheck: func(t *testing.T, filename string) {
+				t.Skip() // TODO: Make all examples work
```

Comment:
Can we re-enable this now?

Reply:
Let's discuss this one.
Some info:
There is a linter that forces this category to be added, but this test [creates a whole new test](https://github.com/grafana/terraform-provider-grafana/pull/1862/files/22be4c46f85aed7cd6303107de418b701df95683#diff-aff89be1f7ad0485c64ec9853c3fdb51f08cea18faf005a10c6220cd6f976110R121-R131) which cannot pass because we are not depending on a _real_ Connections API/access policy token/real metrics endpoint scrape URL.
So I have matched the t.Skip() of all the tests in other categories in this file. The resource test `TestAcc_MetricsEndpointScrapeJob` still runs elsewhere in the CI.

Reply:
Discussed with Matt, we've agreed to t.Skip() as most of the other categories have also done. I've updated the comment to explain why.